.travis.yml

```diff
@@ -22,3 +21,9 @@ matrix:
     - jruby-head
   gemfile:
     - gemfiles/RailsHeadGemfile
+
+matrix:
```

Comment:
why are you redefining `matrix`? Is this supported?

Reply:
I uploaded the PR and noticed the Travis build had failed when using `jruby-head`, so I extracted that onto another matrix, which it is not working so I am currently checking how to do this properly